converter/templates/base.html

```diff
@@ -30,7 +30,7 @@
 
     <footer class="text-center nonselect">
         {% block footer %}
-            <small> &copy; 2020 <a href="http://google.com" title="by Lucien">Lucien</a>
+            <small> &copy; 2020 <a href="https://github.com/lucien386/MusicalKits_Converter" title="by Lucien">Lucien</a>
```

Comment:
Hmm, I wonder whether you want to use a personal website instead of directing people to the source code, though this is open-source project

Reply:
yeah... That's right, I should save it to my personal website.